src/server.js

```diff
@@ -1,26 +1,29 @@
 import React from 'react'
 import { flush } from './style'
 
-export default function flushToReact() {
+export default function flushToReact(nonce = undefined) {
   return flush().map(args => {
     const id = args[0]
     const css = args[1]
     return React.createElement('style', {
       id: `__${id}`,
       // Avoid warnings upon render with a key
       key: `__${id}`,
+      nonce,
       dangerouslySetInnerHTML: {
         __html: css
       }
     })
   })
 }
 
-export function flushToHTML() {
+export function flushToHTML(nonce = undefined) {
```

Comment:
`= undefined` is redundant, you can remove it.

Reply:
can we make the arguments an object? This will allow us to add more options in the future.

```js
function flushToHTML(options = {}) {
  //   options.nonce 
}
```